core/src/com/po/fuck/view/Renderer.java

```diff
@@ -0,0 +1,117 @@
+package com.po.fuck.view;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.utils.ScreenUtils;
+import com.po.fuck.model.Constants;
+import com.po.fuck.model.collections.DrawableCollection;
+
+/**
+ * Class responsible for rendering game objects.
+ */
+public class Renderer {
+    SpriteBatch batch;
+
+    private OrthographicCamera camera;
+
+    static Map<Class<?>, ClassDrawer<?> > drawers = new HashMap<>();
+    static Map<Class<?>, List<Sprite>> sprites = new HashMap<>();
+
+    static {
+        ClassDrawer.initializeDrawers();
+        ClassDrawer.initializeSprites();
+    }
+
+    public Renderer(){
+        camera = new OrthographicCamera();
+        camera.setToOrtho(false, Constants.GAME_WIDTH, Constants.GAME_HEIGHT);
+        batch = new SpriteBatch();
+    }
+
+    float timeElapsed = 0;
+
+    /**
+     * Renders the provided DrawableCollection.
+     *
+     * @param drawableCollection the collection of drawable objects to render
+     */
+    public void render(float delta, DrawableCollection drawableCollection){
+        timeElapsed += delta;
+
+        ScreenUtils.clear(0, 0, 0, 1);
+        camera.update();
+        batch.setProjectionMatrix(camera.combined);
+
+        ClassDrawer.initializeLaserBeamSprites(timeElapsed);
```

Comment:
What ?

Reply:
Unfortunately, we need to have updated GIF animations